_src/app/Domains.js

```diff
@@ -67,29 +65,20 @@ define([
             //      fires when a select is changed to the "other" option
             console.log('app/Domains:onOtherSelected', arguments);
 
-            var existingOptions = [];
-
-            array.forEach(select.children, function (option) {
-                if (option.value !== '' && option.value !== this.otherTxt) {
-                    existingOptions.push({
-                        code: option.value,
-                        name: option.innerHTML
-                    });
-                }
-            }, this);
+            var existingOptions = Array.from(select.children).map(option => option.value);
```

Comment:
This might need to be polyfilled if people are using IE at all.

Reply:
Ah, I thought that Babel would take care of this one for me. I'm not too worried about supporting IE at this point for this project.